agent-install/agent-install.sh

```diff
@@ -1352,6 +1352,21 @@ function check_node_state() {
 	log_debug "check_node_state() end"
 }
 
+# removes agent-install files and deb packages after successful install
```

Comment:
This is a nit, but the comment says remove files **after** the install, but it looks like you are using this function to remove old files **before** the current install. (Which is fine, just correct the comment.)

Reply:
Argh! Totally forgot to change that after moving it to before... Done!